documentation/feature/laser_selfdesignate.md

```diff
@@ -6,21 +6,10 @@ group: feature
 parent: wiki
 ---
 
-## Overview
+## 1. Overview
 
-### Sub-feature 1
-Short description of sub-feature 1.
+Allows gunners to lase their own targets more informations about it's use in fcs.
```

Comment:
"Allows gunners to lase their own targets. See [FCS](http://ace3mod.com/wiki/feature/fcs.html) document for more information."